deploy/runtools/run_farm_deploy_managers.py

```diff
@@ -172,6 +323,10 @@ def copy_sim_slot_infrastructure(self, slotno: int) -> None:
             run(f"mkdir -p {remote_sim_rsync_dir}")
 
             files_to_copy = serv.get_required_files_local_paths()
+
+            # append any URI paths to the end of this list
```

Comment:
No. Each InstanceDeployManager object maintains a member, uri_list, which is a list of URI that need to be copied. This is not shared. Neighbor URI's are not copied over. Is there something I can change to make this more clear?

Reply:
I think I was confused by the singular `get_local_uri_path`.  If it was named `get_local_uri_paths`, it might have helped.  I just wanted to make sure you weren't copying the whole cache by passing rsync a single directory name to copy.